### ISSUE TITLE
Automatic email error reporting broken?

### ISSUE THREAD

**Issue by bernt-matthias** [galaxyproject/galaxy#5964]
I tried to enable automatic email error reporting in `error_report.yml` which resulted in an error on my dev instance:

```
Bug report plugin <galaxy.tools.error_reports.plugins.email.EmailPlugin object at 0x7fda3180f150> generated response ("An error occurred sending the report by email: 'thread._local' object has no attribute 'mapper'", 'danger')
```

Manual error reporting also did not work anymore (the button is still present but has no effect .. except for showing an empty page).

**Comment by hexylena** [comment]
Ugh, seems to still be a bug, sorry @bernt-matthias. For me the bug report page works and is filled, just it fails to send email. Not clear why. I haven't really used the email configuration enough with this.

**Comment by bernt-matthias** [comment]
Forgot about this one. Seems that I opened nearly the same: https://github.com/galaxyproject/galaxy/issues/8444 there I also debugged a bit in the browser and it seems to be (at least in parts) a java script issue. 

Don't know if the output of the Bug report plugin is of any value here. Otherwise we can close this one here. Since I disabled error_report.yml (i.e. now I just have the default sample file in the config dir) I don't see this message anymore... but bug email is still dysfunctional.
